lib/seattleflu/db/cli/command/reportable_conditions.py

```diff
@@ -94,8 +94,12 @@ def webhook(suffix):
             with db.savepoint(f"reportable condition presence_absence_id {record.id}"):
                 LOG.info(f"Processing reportable condition, presence_absence_id «{record.id}»")
 
+                if not record.site:
+                    LOG.info(f"No site found for presence_absence_id «{record.id}». " +
+                        "Inferring site from manifest data.")
+
                 url = SLACK_WEBHOOK_REPORTING_GENERAL \
-                    if record.site not in childrens_sites \
+                    if record.site not in childrens_sites or record.sheet != 'SCH' \
```

Comment:
Unless I'm turned around myself, I'm pretty sure this logic is backwards. `or` should be `and`.

Reply:
Ah, you're right. I tried to make the most minimal change, but negative logic always gets me.